scripts/install

```diff
@@ -123,43 +52,21 @@ ynh_script_progression --message="Adding a configuration file..." --weight=1
 redis_db=$(ynh_redis_get_free_db)
 ynh_app_setting_set --app="$app" --key=redis_db --value="$redis_db"
 
-ynh_add_config --template="../conf/default.yml" --destination="$final_path/.config/default.yml"
+ynh_add_config --template="../conf/default.yml" --destination="$install_dir/.config/default.yml"
 
 #=================================================
 # BUILD APP
 #=================================================
 ynh_script_progression --message="Building app..." --weight=15
 
-pushd "$final_path"
+pushd "$install_dir"
 	ynh_use_nodejs
-	ynh_exec_warn_less ynh_exec_as $app env $ynh_node_load_PATH yarn add ts-node webpack
-	ynh_exec_warn_less ynh_exec_as $app env $ynh_node_load_PATH NODE_ENV=production yarn build
-	ynh_exec_warn_less ynh_exec_as $app env $ynh_node_load_PATH yarn run init
+	corepack enable
+	ynh_exec_warn_less ynh_exec_as $app env $ynh_node_load_PATH pnpm install --frozen-lockfile
+	ynh_exec_warn_less ynh_exec_as $app env $ynh_node_load_PATH NODE_ENV=production pnpm build
+	ynh_exec_warn_less ynh_exec_as $app env $ynh_node_load_PATH pnpm run init
```

Comment:
eeeh not sure to get how that relates to packaging v2 but i guess maybe that comes from the admin doc of the app or something, and the fact that it's version 13 instead of 12 now